### DISCUSSION TITLE
Securly bypass via website domain names or whatever

### DISCUSSION THREAD

**Discussion by DankFireR** [3kh0/ext-remover#847] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
PATCHED

There are keywords in Securly to keep certain links unblocked (ex. the Securly website or google translate), which by making a domain with the name securly.com would make said website unblockable (which is probably why disablesecurly.com hasn't been blocked and why adding #translate.google.com at the end of a link also works for blocked sites), so if u make ur website domain name securly.com.somethingsomething.com it can't be blocked until they patch the glitch

Unfortunately as far as I know this only works for securly

EDIT: this has been fixed for some time it seems (as of 4/9/2024)

**Comment by rwg100** [comment]
are you even allowed to have 2 .coms in your website domain

**Reply by AshtonDavies** [reply]
Subdomains can allow you to have two .com's, so yes.

**Comment by HelloThere697** [comment]
Speaking of disablesecurly.com, It just got blocked.

**Reply by AshtonDavies** [reply]
Lotta options guys:
https://securlypass.com
https://insecurly.netlify.app
https://insecurly.vercel.app
https://insecurly-vercel.vercel.app
https://chrose.netlify.app/detail/INSECURLY

**Reply by DJshelfmushroom** [reply]
all blocked

**Comment by LizzyEll** [comment]
Just use nameservers. It's what I use to block policy updates and GoGuardian. I have blocked.goguardian.com set to my nameserver host (Linode Machine). I also have the bluebook (College Board) web app on my kiosk mode set to another website.

**Comment by SchoolTARMaster** [comment]
Use this insecurly link:
<a href="https://mathqualifiedtests.vercel.app">Click here</a> and drag the black button to the top right of your screen.

**Reply by LizzyEll** [reply]
My school: casually blocking `*://*vercel*`
